src/renderers/sigma.renderers.canvas.js

```diff
@@ -402,11 +396,9 @@
    * @return {sigma.renderers.canvas} Returns the instance itself.
    */
   sigma.renderers.canvas.prototype.clear = function() {
```

Comment:
I remember checking which method was the fastest. I am sure that now the `clearRect` wins in modern browsers. However did you observe perf drops in older ones?